themes/theme-b2b/src/components/Table.styles.ts

```diff
@@ -14,6 +14,7 @@ export const Table: ThemeComponent<'Table'> = {
         variant: {
           compact: 'py-2',
```

Comment:
should we move `compact` and `expanded` to `size` then? Otherwehise it is not possible to make a compact bordered table?